react/server.js

```diff
@@ -0,0 +1,256 @@
+// Express server previewing how to best interact with common Pelm API endpoints
+import express from 'express';
+import fetch, { Headers } from 'node-fetch';
+import dotenv from 'dotenv';
+import session from 'express-session'
+const app = express()
+const port = 3001
+dotenv.config()
+
+// Enviornment variables stored in the .env file you created
+const PELM_CLIENT_ID = process.env.PELM_CLIENT_ID
+const PELM_SECRET = process.env.PELM_SECRET
+const USER_ID = process.env.USER_ID
+
+
+app.use(
+  // Here we are using session in place of a database
+  // Instead use your own database to store values like the Authorization_Token
```

Comment:
suggestion

In this example, we use `session` in lieu of a database for convenience.

Reply:
Switched with line 17